proxy/coco/src/peer/announcement.rs

```diff
@@ -71,7 +70,20 @@ pub async fn build(peer: &Peer<BoxedSigner>) -> Result<Updates, Error> {
                     for (head, hash) in &refs.heads {
```

Comment:
We could do `refs.iter_categories` so that we announce all the refs in the future

Reply:
Yes, I would say so. We haven't seen this happening because updates of `Person` or `Project` haven't been implemented in Upstream, but I would think that the network should see these so followers can get updates and maintainers can resolve updates :)